### ISSUE TITLE
Unhandled rejection TypeError: this.currentHook.endStep is not a function 

### ISSUE THREAD

**Issue by violant** [Shelex/cypress-allure-plugin#40]
**Describe the bug**
I got Allure exception: " Unhandled rejection TypeError: this.currentHook.endStep is not a function " after Before each and before first test in spec.
 `Unhandled rejection TypeError: this.currentHook.endStep is not a function
    at AllureReporter.endHook (http://10.40.67.32/__cypress/tests?p=cypress\support\index.js:35627:30)
    at AllureReporter.failTestCase (http://10.40.67.32/__cypress/tests?p=cypress\support\index.js:35544:18)
    at Runner.eval (http://10.40.67.32/__cypress/tests?p=cypress\support\index.js:35125:31)
From previous event:
    at onNext (http://10.40.67.32/__cypress/runner/cypress_runner.js:173630:20)
    at done (http://10.40.67.32/__cypress/runner/cypress_runner.js:103365:5)
    at http://10.40.67.32/__cypress/runner/cypress_runner.js:103430:11
From previous event:
    at callFn (http://10.40.67.32/__cypress/runner/cypress_runner.js:103428:14)
    at Hook.../driver/node_modules/mocha/lib/runnable.js.Runnable.run (http://10.40.67.32/__cypress/runner/cypress_runner.js:103412:7)
    at http://10.40.67.32/__cypress/runner/cypress_runner.js:173679:28
From previous event:
    at Object.onRunnableRun (http://10.40.67.32/__cypress/runner/cypress_runner.js:173667:20)
    at $Cypress.action (http://10.40.67.32/__cypress/runner/cypress_runner.js:164430:61)
    at Hook.Runnable.run (http://10.40.67.32/__cypress/runner/cypress_runner.js:172280:13)
    at next (http://10.40.67.32/__cypress/runner/cypress_runner.js:103927:10)
    at http://10.40.67.32/__cypress/runner/cypress_runner.js:103966:7
    at next (http://10.40.67.32/__cypress/runner/cypress_runner.js:173603:16)
    at http://10.40.67.32/__cypress/runner/cypress_runner.js:173624:11`

```
//--------------------------spec----------------------------
beforeEach('all', () => {

});

describe("suite", () => {
    it(test, () => {
        cy.log('test new');
        expect(true).to.be.true;
    });


});
```

Cypress version 4.12.1
 cypress-allure-plugin version 2.0.1

DevTools console output is attached.

[console_error_full.txt](https://github.com/Shelex/cypress-allure-plugin/files/5836081/console_error_full.txt)
[console_error.txt](https://github.com/Shelex/cypress-allure-plugin/files/5836082/console_error.txt)

**Comment by Shelex** [comment]
@violant issue should be resolved in v2.0.3. Could you please confirm on your side?

**Comment by violant** [comment]
> 
> 
> @violant issue should be resolved in v2.0.3. Could you please confirm on your side?

Fixed
